icechunk-python/pyproject.toml

```diff
@@ -39,7 +39,7 @@ test = [
   "ruff",
   "dask>=2024.11.0",
   "distributed>=2024.11.0",
-  "xarray@git+https://github.com/pydata/xarray.git@main",
+  "xarray>=2025.01.1",
```

Comment:
same question

Reply:
presumably we want to support xarray forever